test/org/sagebionetworks/bridge/play/controllers/SharedModuleMetadataControllerTest.java

```diff
@@ -120,9 +120,6 @@ public void byIdAndVersion() throws Exception {
         Result result = controller.getMetadataByIdAndVersion(MODULE_ID, MODULE_VERSION);
         assertEquals(200, result.status());
         assertMetadataInResult(result);
-
-        // validate permissions
-        verify(controller).getAuthenticatedSession(Roles.DEVELOPER);
```

Comment:
Regarding non-authenticated calls... yes, through ApiClientProvider, not ClientManager (the class I wrote to make working with sessions easier).